index.html

```diff
@@ -20,19 +20,137 @@
       </ul>
     </nav>
   </header>
+
   <main>
     <section id="introduction-section">
-      <h1 class="introduction-name">I’m Hernán.<br>Glad to see you!</h1>
-      <p class="introduction-paragraph">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <h1 class="introduction-name principal-margin">I’m Hernán.<br>Glad to see you!</h1>
+      <p class="introduction-paragraph principal-margin">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
       <h2 class="lets-connect">LET'S CONNECT</h2>
-      <ul class="social-icons">
+      <ul class="social-icons social-icons-margin-bottom-presentation">
         <li><a href="https://twitter.com/HFG__43"><img src="Images_Background/Twitter.svg" alt="Twitter Icon"></a></li>
         <li><a href="https://www.linkedin.com/in/hern%C3%A1n-g%C3%BCemes-a440591b/"><img src="Images_Background/Linkedin icon.svg" alt="LinkedIn Icon"></a></li>
         <li><a href="#"><img src="Images_Background/M.svg" alt="M Icon"></a></li>
         <li><a href="https://github.com/HFG43"><img src="Images_Background/GitHub.svg" alt="GitHub Icon"></a></li>
         <li><a href="#"><img src="Images_Background/Fingers.svg" alt="Fingers Icon"></a></li>
       </ul>
     </section>
+
+    <section id="card-works-container">
+      <article id="card-work-1" class="mobile-card">
+        <img class="card-snapshot" src="Images_Background/Snapshoot Portfolio 1_Orange.svg" alt="Project 1 Snapshot">
+        <section class="project-description-block">
+          <h2 class="card-title">Tonic</h2>
+          <ul class="project-role-detail">
+            <li class="project-role-client ">CANOPY</li>
+            <li><i class="fa-solid fa-circle fa-2xs project-role-counter"></i></li>
+            <li>Back End Dev</li>
+            <li><i class="fa-solid fa-circle fa-2xs project-role-counter"></i></li>
+            <li>2015</li>
+          </ul>
+          <p class="project-description-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="project-language-list">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div class="button-container">
+            <button class="see-project-button">See project</button>
+          </div> 
+        </section>
+      </article>
+      <article id="card-work-2" class="mobile-card">
+        <img class="card-snapshot" src="Images_Background/Snapshoot Portfolio 2_Availability.svg" alt="Project 2 Snapshot">
+        <section class="project-description-block">
```

Comment:
[OPTIONAL] I would suggest to use a div here rather than a section tag, as section is used to represent a generic part of the page and since it is just a division a div would be better here.